### PR TITLE
[Moodle] Add guide for Moodle

### PR DESCRIPTION
**Pull request by proohit**
Hello there, i've finally written the guide for Moodle. Now anyone can setup an own Moodle within minutes via Uberspace 🚀 .

Closes: #597

### CODE REVIEW

File: source/guide_moodle.rst [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+4. For the Database Settings, we enter our information:
+  
+  1. ``Database name``: ``isabell_moodle``
+  2. ``Database user``: ``isabell``
+  3. ``Database password``: ``MySuperSecretPassword``
+  4. ``Database port``: ``3306``
+
+5. Read the Copyright notice and only accept if you truly do :)
+6. The checks should now show all OK. At the end of the page, we click on ``Continue``. This could take a while to finish.
+7. You should be redirected to the installation status screen. If you're not being redirected automatically, just wait a little bit. Try refreshing the exact site. It has to be an URL something like https://isabell.uber.space/admin/index.php?cache=0&agreelicense=1&confirmrelease=1&lang=en . At this point, it may be possible that a message like "Site is being upgraded, please retry later" pops up after reloading. Just try reloading after some minutes when this occurs.
```
Comment (luto) [inline source/guide_moodle.rst:122]: While trying this guide I had major troubles with this. Even after several reloads and a bunch of waiting, moodle wouldn't show up. Once it finally did the 2nd stage of the setup process wouldn't continue until I deleted a database table it complained about. In the end it was working, but that's not something we can document and support for users.

We'll do a few performance upgrade in the following weeks. Once those are done, I'll take another look. Sorry about the delay.

**Comment by proohit** [comment]
Thanks for reviewing. I was thinking that it somehow was related to my setup but it seems that it is reproducable to some extent.

I do not know explicitly when everything is working fine or not. Sometimes the setup proceeds completely fine (e.g. my very first time setting it up).

I get the feeling that installation of modules that are needed heavily affect the performance including the web server.  I will also further look into this but I'd be happy about assistance. Cheers and keep healthy.

**Comment by luto** [comment]
mysql is much faster now, so this is ripe for a re-test :)

**Comment by proohit** [comment]
> mysql is much faster now, so this is ripe for a re-test :)

Wow nice! This made a huge a difference!

I tried the setup again with a fresh instance of uberspace and faced no issues!

**Comment by luto** [comment]
Perfekt! :tada: :tada: can you please remove the "At this point, it may be possible that a message like "Site is being upgraded, please retry later" pops up (...)" wording then? Afterwards I am more than happy to finally merge this :)

**Comment by luto** [comment]
Thanks! :)

**Pull request merged**